README.md

```diff
@@ -1,52 +1,239 @@
-<h1 align="center">Atlas</h1>
-<p align="center">Explore, label, search and share massive datasets in your web browser.</p>
+<p align="center">
+      <img src="./assets/NOMIC_x2.png" alt="Nomic Logo" style="width: 40%; max-height: 160px; width: auto">
```

Comment:
Do not use the green logo, use a black one.

Reply:
Vincent has pointed out to not use the black logo with white or transparent background because of Github dark mode.

Reply:
lets drop the logo altogether